app.py

```
@@ -81,26 +81,51 @@ def generate_questions(env_variables):
 
     env_variables = parse_env_file(env_content)
-    print(env_variables,'hhh')
     session['env_variables'] = env_variables
     session['answers'] = {}
     session['index'] = 0  # Track the current question index
-    
+
     # Randomly select MAX_QUESTIONS number of keys to ask
     keys = list(env_variables.keys())
     selected_keys = random.sample(keys, app.config['MAX_QUESTIONS'])
     selected_env_variables = {key: env_variables[key] for key in selected_keys}
```

Ruff (0.8.2)

Local variable `selected_env_variables` is assigned to but never used

Remove assignment to unused variable `selected_env_variables`

(F841)